rocks-lib/src/config/mod.rs

```diff
@@ -145,7 +166,7 @@ impl Default for Config {
             only_server: None,
             only_sources: None,
             namespace: "".into(),
-            lua_dir: None,
+            lua_dir: Config::get_default_cache_path().unwrap().join("lua"),
```

Comment:
note(non-blocking): I'm not sure if a cache is the right location to install lua.
Perhaps a data directory (e.g. in `get_default_data_path()`) would be more suitable?